### DISCUSSION TITLE
Need help for my mobile server

### DISCUSSION THREAD

**Discussion by hawksmith505** [Pryaxis/TShock#2152] [Q&A]
After for a while running my server on mobile, players with the tl pro application thing starts to spawn random monsters on my server, any solution from preventing them to use that application. really need help.

**Comment by hawksmith505** [comment]
banning them can't help because they can still make new account

**Reply by EMREOYUN** [reply]
Current bouncer is kindly not working. You can use plugins like DieMob.
